deploy-manage/deploy/self-managed/_snippets/enable-auto-indices.md

```diff
@@ -0,0 +1,9 @@
+Some features automatically create indices within {{es}}. By default, {{es}} is configured to allow automatic index creation, and no additional steps are required. However, if you have disabled automatic index creation in {{es}}, you must configure [`action.auto_create_index`](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-create) in `elasticsearch.yml` to allow features to create the following indices:
```

Comment:
The link of `action.auto_create_index` doesn't feel correct. It should point to wherever https://www.elastic.co/guide/en/elasticsearch/reference/current/index-management-settings.html is.

Also in the text `if you have disabled automatic index creation` we should offer a link, as the reader may think `how that could be done`? as it's not obvious. The best page is probably https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#index-creation (wherever that is :) ).

Reply:
It's also important to clarify that all that section is only needed if the user has explicitly changed `action.auto_create_index` from its default value.

I would suggest to add in a banner that statement (that section is only needed if...) instead of the banner for logstash and beats.